src/CameraControls.ts

```diff
@@ -153,6 +153,11 @@ export class CameraControls extends EventDispatcher {
 
 		super();
 
+		if (typeof THREE === 'undefined') { //Chech if the user has installed THREE
+			console.error("Three.js is undefined. You must first run CameraControls.install({ THREE: THREE }). Check the docs for further information.")
+			return; //Stop the execution as there will be errors
```

Comment:
Could I omit `return` here?
Without initializing properties (such as `this._camera` and others ) causes TS error 😢.
```suggestion
```

Reply:
For sure!